fontforge/python.c

```diff
@@ -221,42 +222,6 @@ static wchar_t *copy_to_wide_string(const char *s) {
 #endif /* NEED_WIDE_CHAR */
 
 
```

Comment:
This entire function was pointless because `PyUnicode_AsUTF8String` sets a more verbose exception if one exists.